app/presenters/document_presenter.rb

```diff
@@ -12,6 +12,8 @@ def to_json
       base_path: document.base_path,
       title: document.title,
       description: document.summary,
+      document_type: document.document_type,
+      schema_name: document.schema_name,
       format: "specialist_document",
```

Comment:
I think you can't specify the `format` if you send `document_type` & `schema_name`.